scarlet/observation.py

```diff
@@ -35,10 +35,16 @@ class Frame():
         PSF in each band
     channels: list of hashable elements
         Names/identifiers of spectral channels
+    dtype: `numpy.dtype`
+        Dtype to represent the data.
     """
-    def __init__(self, shape, wcs=None, psfs=None, channels=None):
+    def __init__(self, shape, wcs=None, psfs=None, channels=None, dtype=np.float32):
         assert len(shape) == 3
         self._shape = tuple(shape)
+
+        if self.Ny != fftpack.next_fast_len(self.Ny) or self.Nx != fftpack.next_fast_len(self.Nx):
+            logger.warning('Shape ({},{}) not fast for FFT. Consider increasing to ({},{}) pixels'.format(self.Ny, self.Nx, fftpack.next_fast_len(self.Ny), fftpack.next_fast_len(self.Nx)))
+
```

Comment:
I might be missing something but I don't think that this is necessary. The shape that we want to be fast is the shape that is `Frame.shape + Observation.psfs.shape`, since we need padding to prevent artifacts in the image. So the user shouldn't really need to care about this at all, since we'll be padding anyway.

Reply:
Hmm, right. Does that mean #111 is a non-issue?

Reply:
Yeah, either we were being more clever last week than I am right now or I wasn't thinking about this properly when that issue was created.